### PR TITLE
Update email validation

### PR DESCRIPTION
**Pull request by ros3bud**
@alexkwolfe, LGTY? 

This presents a solution to the confusing nature of presenting email.corrected as the same as email.original when no correction was made.

- Return null when no correction is made.
- Updated language and tests to reflect this change.
- Updated response parsing to always store webbula threat in event.threat, even when unknown, valid or clean.

### CODE REVIEW

File: src/email.coffee [hunk 1]
```diff
@@ -93,6 +92,11 @@ badEmail = (value) ->
    return false
   else return true
 
+parseCorrected = (value) ->
+  return null unless value? 
+  return value.email if value.corrected?
+  return null
```
Comment (alexkwolfe) [inline src/email.coffee]: Could this be simplified to the below?

```coffeescript
parseCorrected = (value) ->
  if value?.corrected? then value.email else null
```

Comment (alexkwolfe) [inline src/email.coffee]: If so, then you could probably just put that statement on line 63 rather than declare a whole new function.

Comment (ros3bud) [inline src/email.coffee]: It looks like that does work.

**Comment by ros3bud** [comment]
@alexkwolfe  ^^

**Comment by alexkwolfe** [comment]
LGTM

**Pull request merged**